app/assets/javascripts/application.js

```diff
@@ -55,3 +55,4 @@
 //= require thredded-custom
 //= require flash_messages_on_xhr
 //= require js.cookie
+//= require i18n/translations
```

Comment:
Is this really necessary as we use js translation only in the admin interface? Delete it otherwise